server/controllers/profile.js

```diff
@@ -1,24 +1,16 @@
+const asyncHandler = require("express-async-handler");
 const Profile = require("../models/Profile");
 const User = require("../models/User");
-const asyncHandler = require("express-async-handler");
 
 // @route POST /profile/edit
 // @desc edit user profile
-// @access Public
+// @access Private
 exports.editProfile = asyncHandler(async (req, res, next) => {
-  const user = await User.findById(req.user.id);
+  const profile = await Profile.findByIdAndUpdate(req.body.id, req.body.changes, { new: true });
```

Comment:
Suggestion: For security reasons only the owner of a profile should be able to update it 😄

Reply:
Made the appropriate changes!